### PR TITLE
GPL-v3 license.

### PR DESCRIPTION
**Pull request by amanabt**
A GPL-v3 license has been added as suggested by @mchandra

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    {one line to give the program's name and a brief idea of what it does.}
+    Copyright (C) {year}  {name of author}
+
```
Comment (mchandra) [inline LICENSE]: You need to fill this. Look at https://github.com/QuazarTech/Bolt/blob/master/LICENCE for an example.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,674 @@
...
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    {project}  Copyright (C) {year}  {fullname}
+    This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
+    This is free software, and you are welcome to redistribute it
+    under certain conditions; type `show c' for details.
+
```
Comment (mchandra) [inline LICENSE]: This too.

Comment (amanabt) [reply LICENSE]: ok

**Comment by amanabt** [comment]
made the changes.

**Pull request merged**